tap_frontapp/__init__.py

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env python3
-
+# pylint: disable=E0602
```

Comment:
can you address the issue that caused this pylint error?
its not a good practice to disable Error/fatal pylint messages.